### PR TITLE
Adds various `badge` icons, rebases `verified` as `badge-check`

### PR DESCRIPTION
**Pull request by karsa-mistmere**
closes #1109 #1073 

Addresses some leftover comments from #416 and renames `verified` to be `badge-check` while also adding its base shape.

Also makes the shape more consistent in size with `circle`.

Also adds https://github.com/feathericons/feather/issues/865 as part of #119

### CODE REVIEW

**Comment by jguddas** [comment]
What about using this version of the outline instead? #1073

File: icons/badge-dollar-sign.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
+  <path d="M16 8h-6a2 2 0 0 0-2 2v0a2 2 0 0 0 2 2h4a2 2 0 0 1 2 2v0a2 2 0 0 1-2 2H8" />
+  <path d="M12 18V6" />
```
Comment (jguddas) [inline icons/badge-dollar-sign.svg:14]:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
  <path d="M16 8h-6a2 2 0 1 0 0 4h4a2 2 0 1 1 0 4H8" />
  <path d="M12 18V6" />
```

Comment (jguddas) [reply icons/badge-dollar-sign.svg:14]: We could also make the dollar smaller `<path d="M15 8h-4.5a2 2 0 1 0 0 4h3a2 2 0 1 1 0 4H9" />` file-question

File: icons/badge-help.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
```
Comment (jguddas) [inline icons/badge-help.svg:12]:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
```

File: icons/badge-info.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
```
Comment (jguddas) [inline icons/badge-info.svg:12]:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
```

File: icons/badge-minus.svg [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
```
Comment (jguddas) [inline icons/badge-minus.svg:12]:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
```

File: icons/badge-minus.svg [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  <line x1="8" x2="16" y1="12" y2="12" />
```
Comment (jguddas) [inline icons/badge-minus.svg:13]: Maybe a smaller minus symbol fits better
```suggestion
  <path d="M9 12h6" />
```
file-question (2)

File: icons/badge-percent.svg [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
```
Comment (jguddas) [inline icons/badge-percent.svg:12]:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
```

File: icons/badge-plus.svg [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
+  <line x1="12" x2="12" y1="8" y2="16" />
+  <line x1="8" x2="16" y1="12" y2="12" />
```
Comment (jguddas) [inline icons/badge-plus.svg:14]: Smaller plus symbol?
```suggestion
  <line x1="12" x2="12" y1="8" y2="16" />
  <line x1="8" x2="16" y1="12" y2="12" />
```
file-question (3)

File: icons/badge-x.svg [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
+  <line x1="15" x2="9" y1="9" y2="15" />
+  <line x1="9" x2="15" y1="9" y2="15" />
```
Comment (jguddas) [inline icons/badge-x.svg:14]: Could also be smaller?

File: icons/circle-dollar-sign.svg [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="10" />
+  <path d="M16 8h-6a2 2 0 0 0-2 2v0a2 2 0 0 0 2 2h4a2 2 0 0 1 2 2v0a2 2 0 0 1-2 2H8" />
```
Comment (jguddas) [inline icons/circle-dollar-sign.svg:13]: Also smaller?

File: icons/badge-alert.svg [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3.852 8.625A3.997 3.997 0 0 1 4.93 4.929a3.997 3.997 0 0 1 3.696-1.077A3.997 3.997 0 0 1 12 2c1.419 0 2.665.739 3.375 1.852a3.997 3.997 0 0 1 3.696 1.077 3.997 3.997 0 0 1 1.077 3.696A3.997 3.997 0 0 1 22 12a3.997 3.997 0 0 1-1.852 3.375 3.997 3.997 0 0 1-1.077 3.696 3.997 3.997 0 0 1-3.696 1.077A3.997 3.997 0 0 1 12 22a3.997 3.997 0 0 1-3.375-1.852 3.997 3.997 0 0 1-3.696-1.077 3.997 3.997 0 0 1-1.077-3.696A3.997 3.997 0 0 1 2 12c0-1.419.739-2.665 1.852-3.375Z" />
```
Comment (jguddas) [inline icons/badge-alert.svg:12]:
```suggestion
  <path d="M3.85 8.63a4 4 0 0 1 4.78-4.78 4 4 0 0 1 6.74 0 4 4 0 0 1 4.78 4.77 4 4 0 0 1 0 6.76 4 4 0 0 1-4.77 4.77 4 4 0 0 1-6.76 0 4 4 0 0 1-4.77-4.77 4 4 0 0 1 0-6.76" />
```

File: icons/badge-minus.svg (hunk 5 above)
Comment (karsa-mistmere) [reply icons/badge-minus.svg:13]: I think it should be the exact same as in all other similar icons (circle, square etc.) as long as the guidelines don't come into play.

File: icons/circle-dollar-sign.svg (hunk 9 above)
Comment (karsa-mistmere) [reply icons/circle-dollar-sign.svg:13]: We very barely abide by the 2px gap as it is, so we definitely can't go any smaller. :D

Comment (jguddas) [reply icons/circle-dollar-sign.svg:13]: You are right, it also has to look good with 3px stroke width.

Comment (jguddas) [inline icons/circle-dollar-sign.svg:13]:
```suggestion
  <path d="M16 8h-6a2 2 0 1 0 0 4h4a2 2 0 1 1 0 4H8" />
```

Comment (karsa-mistmere) [reply icons/circle-dollar-sign.svg:13]: Working your magic as usual. :)

Comment (jguddas) [reply icons/circle-dollar-sign.svg]: The magic is still WIP, but I pushed it to the edit page PR, so you can have a look if you like.
https://github.com/lucide-icons/lucide/blob/d8e95b327b30086085e53d2181974e24170a31db/site/src/pages/api/edit/optimize.ts#L628-L641
- **elementsToPath:** Consistently converts `line`, `polygon`, `polyline` to `path`.
- **mergePaths**: Consistently combines shapes that are connected into a single path, unless they are closed.
- **mergeArcs**: Sadly a bit flaky. (the magic that fixed the problem above)
- **segmentsToArc**: Also, a bit flaky (needed for the next step and generally nicer to read).
- **pathsToElement**: Consistently converts paths to (`rect` and `circle`), `ellipsis` is out of scope.
- **optimizeRect**: Removes useless properties from rect.
- **removeUselessClosingSegments**: Removes zero width `Z` segments.
- **smartClose**: Replaces the last `H`, `V` or `L` segment with `Z` when path is circular.
- **snapToGrid**: This does some rounding.
- **fixDots**: This replaces zero width paths for figma.
- **svgo**: Convert paths to a mix out of absolute and relative.
- **format**: Sorts arguments and paths. Also removes duplicates.

Comment (jguddas) [reply icons/circle-dollar-sign.svg]: Here is a fun example:

https://lucide-site-5dko05lj3-jguddas.vercel.app/edit/lucide-icons/lucide/plug-zap?PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJtMTMgMi0yIDIuNWgzTDEyIDciIC8+CiAgPHBhdGggZD0iTTEyIDIydi0zIiAvPgogIDxwYXRoIGQ9Ik0xMCAxM3YtMi41IiAvPgogIDxwYXRoIGQ9Ik0xMCAxMi41di0yIiAvPgogIDxwYXRoIGQ9Ik0xNCAxMi41di0yIiAvPgogIDxwYXRoIGQ9Ik0xNiAxNWEyIDIgMCAwIDAtMi0yaC00YTIgMiAwIDAgMC0yIDJ2MmEyIDIgMCAwIDAgMiAyaDRhMiAyIDAgMCAwIDItMnYtMnoiIC8+Cjwvc3ZnPg==

**Review by ericfennis — COMMENTED** [review]
Hmm I'm not sure with increasing the size of the badge outline.
I think it breaks a bit the balance in the icons
I placed side by side with twitter icon, and I think the original `verified` in lucide. looks way better. 
image

**Comment by karsa-mistmere** [comment]
@ericfennis: well, in comparison to Twitter maybe, but in comparison to our own `circle-check` icon, I think it's the other way around (this was the trigger of this PR in the first place):

image

**Comment by danielbayley** [comment]
> in comparison to our own circle-check icon

I think the circle is too big in that instance 😅

**Comment by karsa-mistmere** [comment]
@jguddas: well, `circle` and `square` are our bases for all other icons. :D

**Comment by karsa-mistmere** [comment]
> I get that… But I think the crux of it is that `circle` is bigger (20x20px) than `square` (18x18px)…

Well, that's the exactly the point. [Squares have a smaller visual weight than circles](https://medium.com/design-bridges/optical-effects-9fca82b4cd9a). 🙂

**Comment by jguddas** [comment]
If anything than it's the square that's to big. `π*10px^2 = 314px`  vs `18x18px = 324px`

**Comment by danielbayley** [comment]
> Squares have a smaller visual weight than circles

Sure, but I mean in terms of trying to fit the same elements nicely inside both is problematic :)

**Comment by karsa-mistmere** [comment]
> If anything than it's the square that's to big. `10px²×π = 314px²` vs `18px×18px = 324px²`

Oh, but you're forgetting the 2px rounding!😄
That will result in `320.56px²`, which is amazingly close considering we're bound to a 24 by 24 pixel grid.

**Comment by jguddas** [comment]
We are not super consistent when we look at the convex hull to compare visual weight.

image

https://observablehq.com/d/fdcccb6b2881ba87

**Review by ericfennis — APPROVED** [review]

**Pull request merged**